ccdproc/tests/test_ccdproc.py

```diff
@@ -619,8 +618,7 @@ def test_catch_transform_wcs_warning():
     def tran(arr):
         return 10 * arr
 
-    with catch_warnings() as w:
-        tran = transform_image(ccd_data, tran)
+    tran = transform_image(ccd_data, tran)
```

Comment:
This was trying to capture a warning issued when the image being transformed has WCS (which we don't transform). Let me know -- if you want me to push the updates for this.

Reply:
I'll see if I can do something. Follow-up commit coming up.